libraries/plugins/peerplays_sidechain/sidechain_net_handler.cpp

```diff
@@ -102,7 +102,7 @@ void sidechain_net_handler::sidechain_event_data_received(const sidechain_event_
    }
 
    // Withdrawal request
-   if ((sed.peerplays_to == gpo.parameters.son_account()) && (sed.sidechain_currency.compare("1.3.0") == 0)) {
+   if ((sed.peerplays_to == gpo.parameters.son_account()) && (sed.sidechain == sidechain_type::peerplays)) {
```

Comment:
You have to track exact asset here, not whole sidechain, because withdrawal can be done only int BTC asset.